prov/rxm/src/rxm.h

```diff
@@ -657,7 +656,8 @@ struct rxm_ep {
 	struct fid_pep 		*msg_pep;
 	struct fid_eq 		*msg_eq;
 	struct fid_cq 		*msg_cq;
-	uint32_t		msg_cq_eagain_count;
+	uint64_t		msg_cq_last_poll;
+	int			msg_cq_fd;
```

Comment:
This seems like an unused variable.

Reply:
Must have come for the ride with one of the merge conflicts. Removed.